OneSignal.Android.Binding/OneSignal.Android.Binding.csproj

```diff
@@ -55,5 +55,8 @@
   <ItemGroup>
     <LibraryProjectZip Include="Jars\onesignal-release.aar" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.12.0.2" />
+  </ItemGroup>
```

Comment:
i require this nuget because some methods and types that are required to customize notifications were referencing  the androidx.core.app.NotificationCompat$Extender interface and without this nuget it was not resolved and all those methods/types were not emitted in the binding library.